docs/core/project-sdk/msbuild-props.md

```diff
@@ -293,6 +294,18 @@ The `CopyLocalLockFileAssemblies` property is useful for plugin projects that ha
 > [!TIP]
 > Alternatively, you can use `dotnet publish` to publish the class library. For more information, see [dotnet publish](../tools/dotnet-publish.md).
 
+### IsPublishable
+
+The `IsPublishable` property allows the `Publish` target to run. This property only affects processes that use *.csproj* files and the `Publish` target, such as the [dotnet publish](../tools/dotnet-publish.md) command. Visual Studio is not affected because Visual Studio uses the `PublishOnly` target.
```

Comment:
```suggestion
The `IsPublishable` property allows the `Publish` target to run. This property only affects processes that use _.\*proj_ files and the `Publish` target, such as the [dotnet publish](../tools/dotnet-publish.md) command. Visual Studio is not affected because Visual Studio uses the `PublishOnly` target.
```